delivery/subscription.mdx

```diff
@@ -83,33 +83,39 @@ If you specify `FULL` instead of the `since_clause`, the subscription cursor sta
 FETCH from cursor function is currently only supported in the PSQL simple query mode. If you are using components like JDBC that default to the extended query mode, please manually set the mode to simple query mode.
```

Comment:
Extended mode is also supported in https://github.com/risingwavelabs/risingwave/pull/17821 starting from `v2.0`